aspnetcore/blazor/fundamentals/logging.md

```diff
@@ -676,7 +676,28 @@ For the `configureLogging` log level value, pass the argument as either the stri
 | <xref:Microsoft.Extensions.Logging.LogLevel.Critical>    | `critical`     | 5               |
 | <xref:Microsoft.Extensions.Logging.LogLevel.None>        | `none`         | 6               |
 
-Example 1: Set the <xref:Microsoft.Extensions.Logging.LogLevel.Information> log level with a string value:
+Example 1: Set the <xref:Microsoft.Extensions.Logging.LogLevel.Information> log level with a string value.
+
+:::moniker range=">= aspnetcore-8.0"
+
+Blazor Web App:
+
+```html
+<script src="{BLAZOR SCRIPT}" autostart="false"></script>
```

Comment:
Yes, those are the problems that the text/examples were trying to overcome. It was made more generic initially to avoid additional versioning.

I'll make updates for ...

* `{PATH PREFIX}/blazor.web.js`
* `{PATH PREFIX}/blazor.server.js`

... on the next commit.

Reply:
On second thought, I don't think it matters because we're not asking the reader to change it. It's not all that relevant. They need to make sure that the `autostart="false"` is there.

Reply:
> On second thought, I don't think it matters because we're not asking the reader to change it. It's not all that relevant. They need to make sure that the autostart="false" is there.

A possible scenario where this could matter is the following: Suppose someone has a .NET 7 Blazor Server app and they upgrade it to .NET 8. They only change the TFM of the project and keep everything else unchanged (including the `blazor.server.js` script). If they refer to the .NET 8 docs, they may incorrectly assume that they need to add an extra `circuit: { ... }` property to the `Blazor.start()` options parameter object, but that won't work because that format is only valid when the `blazor.web.js` script is used.

Alternatively, they might switch the script to `blazor.web.js` without realizing that this is what distinguishes a Blazor Web App from a Blazor Server app. They may still think of their app as a "Blazor Server" app (maybe because they still use only server interactivity) and consult the Blazor Server section of the docs.

I guess I just want to make sure we're clear that the structure of each options object does not depend on the TFM or the template that was used to create the project initially. It depends on whether `blazor.{server|webassembly}.js` or `blazor.web.js` is used.

I'm not trying to be insistent on using explicit script names in the docs, so if you think it's clear enough then that's fine with me! And your new "Location of the Blazor script" section might be enough to clarify the correlation between Blazor app "type" and which script should be used 🙂

Reply:
> they might switch the script to blazor.web.js without realizing that this is what distinguishes a Blazor Web App from a Blazor Server app.

That should be documented better generally. We don't currently say it for the new section on the Blazor script or in migration/integration content. I'll follow-up with more work to try and fix the problem.

Reply:
Thanks! One thing I'd like to clarify about my previous quote though:

> they might switch the script to blazor.web.js without realizing that this is what distinguishes a Blazor Web App from a Blazor Server app.

I'm not trying to say that we _should_ document this as being the difference. I'm more just saying that the current version of the document sometimes uses the phrase "Blazor Web App" when it really seems to mean "Apps that use `blazor.web.js`". It might be a little tricky to come up with a single definition for what quantifies a "Blazor Web App" because there are lots of other things that are different between the new "Blazor Web App" and the old "Blazor Server App" templates (e.g., different startup code, having an `App.razor` vs. `_Host.cshtml`, etc.).